Go.YAML

```
@@ -0,0 +1,188 @@
+    - name: Check PR Title for Semantic Compliance
+      uses: amannn/action-semantic-pull-request@v5.5.3
+      with:
+        githubBaseUrl: https://api.github.com
+
+    - name: Add Sticky Pull Request Comment
+      uses: marocchino/sticky-pull-request-comment@v2
+      with:
+        header: pr-title-lint-error
+        message: |
+          Hey there and thank you for opening this pull request! 👋🏼
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)